src/libmongoc/tests/test-mongoc-handshake.c

```diff
@@ -183,7 +183,7 @@ test_mongoc_handshake_data_append_success (void)
    mock_server_run (server);
    uri = mongoc_uri_copy (mock_server_get_uri (server));
    mongoc_uri_set_option_as_utf8 (uri, MONGOC_URI_APPNAME, "testapp");
-   pool = test_framework_client_pool_new_from_uri (uri, NULL);
+   pool = mongoc_client_pool_new (uri);
```

Comment:
Please update this file to use `test_framework_client_pool_new_from_uri` and replace the calls to `mock_server_receives_legacy_hello` with `mock_server_receives_any_hello`.

Reply:
+1